deps/zlib/zconf.h.included

```diff
@@ -0,0 +1,562 @@
+/* zconf.h -- configuration of the zlib compression library
```

Comment:
what is this dependency?

Reply:
@MoLow For some reason, my VSCode generated the file automatically when building MAKE. Will remove it.

Reply:
File has been removed and the original has been restored.